### ISSUE TITLE
TypeError in tf_text_graph_ssd.py script

### ISSUE THREAD

**Issue by skaldesh** [opencv/opencv#14993]
##### System information (version)

- OpenCV => 4.1
- Operating System / Platform => Arch-Linux 64 Bit, Kernel 5.1.15-arch1-1-ARCH
- Compiler => cmake + ninja

##### Detailed description

I am trying to use the `tf_text_graph_ssd.py` script to generate a `.pbtxt` textgraph file from a tensorflow `frozen_inference_graph.pb` file.
At first, I got the same exception that was mentioned in [this issue](https://github.com/opencv/opencv/issues/11560#issuecomment-391047037) and I applied the fix. But then I got another exception:  
```python
Traceback (most recent call last):
  File "/scripts/tf_text_graph_ssd.py", line 377, in <module>
    createSSDGraph(args.input, args.config, args.output)
  File "/scripts/tf_text_graph_ssd.py", line 261, in createSSDGraph
    addConstNode('concat/axis_flatten', [-1], graph_def)
  File "/scripts/tf_text_graph_common.py", line 114, in addConstNode
    graph_def.node.extend([node])
TypeError: Not a cmessage
```

##### Steps to reproduce
0. (I can not provide my dataset that I used for training, sorry)
1. Use the latest tensorflow models+scripts from [master](https://github.com/tensorflow/models)
2. Use the latest `tf_text_graph` scripts from this repo.
3. I trained the ssdlite mobilenet v2, here is my [dnn.cfg](https://github.com/opencv/opencv/files/3367446/dnn.txt)
4.  I used this [pre-trained-model](http://download.tensorflow.org/models/object_detection/ssdlite_mobilenet_v2_coco_2018_05_09.tar.gz)
5. You can create the `frozen_inference_graph.pb` file yourself by running
```python
python /scripts/models/research/object_detection/model_main.py \
        --pipeline_config_path="dnn.cfg" \
        --model_dir="models"
python /scripts/models/research/object_detection/export_inference_graph.py \
        --input_type=image_tensor \
        --pipeline_config_path="dnn.cfg" \
        --trained_checkpoint_prefix="models/model.ckpt-100" \
        --output_directory="export"
```
or you use my [frozen_inference_graph.pb](https://github.com/opencv/opencv/files/3367447/frozen_inference_graph.txt)
6.  Then use this updated [tf_text_graph_ssd.py](https://github.com/opencv/opencv/files/3367556/tf_text_graph_ssd.txt) (I already applied the fix from https://github.com/opencv/opencv/issues/11560#issuecomment-391047037) and execute this command:
```python
python tf_text_graph_ssd.py \
        --input "export/frozen_inference_graph.pb" \
        --config "dnn.cfg" \
        --output "model.pbtxt"
```
7. You will receive the stacktrace mentioned above

**Comment by skaldesh** [comment]
Solved with #15176, I am closing this usse once it is merged into master
